examples/firmware_update_example/src/xi_sft_client.c

```diff
@@ -135,7 +139,7 @@ void on_sft_message( xi_context_handle_t in_context_handle,
                                  params->message.temporary_payload_data_length );
 
             /* Figure out what the packet type is and then call the appropriate parser */
-            cb = cn_cbor_decode( payload_data, payload_length, 0 );
+            cb = cn_cbor_decode( payload_data, payload_length CBOR_CONTEXT_PARAM, NULL );

```

Comment:
Could you briefly leave a reply to this message as to what the difference is between this CBOR_CONTEXT_PARAM and the COMMA version?